docs/api.md

```diff
@@ -2773,6 +2773,16 @@ Ractive({
 	delimiters: [ '//', '\\' ]
 });
 ```
+Global data attributes may be specified:
+```js
+Ractive.defaults.data.people = [{id:4, name:'Fred'},{id:5, name:'Wilma'},...]
+
+//or alterntively, here using underscore's extend function
+_.extend(Ractive.defaults.data,{people : [{id:4, name:'Fred'},{id:5, name:'Wilma'},...],
+                                title : 'Flintstones',
+                                producer : 'Hanna-Barbera'})
+```
+The data attributes and values are then accessible in all components.
```

Comment:
I'd add a note that updating a defaulted attribute won't cause invalidation in any instances that happen to reference it.

Reply:
I would also skip mentioning underscore. Gives the impression that it's required. `Object.assign` would be fine.

Reply:
@evs-chris happy to add your suggestion, but I don't understand it! Is there a validation concept in Ractive instances? Also not sure what update-when-referenced behaviour you are thinking of that will not occur with `Ractive.defaults.data`.

@fskreuz good point. What are the dependencies required when `Object.assign` is invoked? Babel? ES6?

Reply:
Sorry, the "won't cause invalidation/updates" bit means that changing the default won't flag any uses as dirty and needing to be updated. Like this:

```js
Ractive.defaults.data.foo = 'bar';
new Ractive({ template: '{{foo}}' }); // bar
Ractive.defaults.data.foo = 'baz';
// above instance still shows bar
```

Ractive actually uses `Object.assign` quite a bit internally, so if the browser doesn't already have one, then it will be polyfilled for you already.